api/envoy/config/metrics/v3/metrics_service.proto

```diff
@@ -25,4 +27,8 @@ message MetricsServiceConfig {
 
   // The upstream gRPC cluster that hosts the metrics service.
   core.v3.GrpcService grpc_service = 1 [(validate.rules).message = {required: true}];
+
+  // If true, counters are reported as the delta between flushing intervals. Otherwise, the current
+  // counter value is reported. Defaults to false.
```

Comment:
Maybe point out that unset implies behavior implied by inline, reference issue.

Reply:
I decided to leave as is because I would be adding a comment about behavior that is not there. But I can update with explanation